dumper-integration-tests/redshift-tests/build.gradle

```diff
@@ -22,28 +19,24 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.codehaus.groovy:groovy-all:3.0.5'
+    implementation 'org.codehaus.groovy:groovy-all:3.0.10'
+    implementation 'com.google.guava:guava:31.1-jre'
     implementation 'org.testng:testng:7.5'
     implementation 'org.slf4j:slf4j-api:2.0.0-alpha5'
     implementation 'org.slf4j:slf4j-jdk14:2.0.0-alpha5'
-    implementation 'com.amazon.redshift:redshift-jdbc42:2.1.0.6'
+    implementation 'com.amazon.redshift:redshift-jdbc42:2.1.0.7'
     implementation 'commons-io:commons-io:2.11.0'
     implementation 'org.apache.commons:commons-collections4:4.4'
-    implementation 'com.google.guava:guava:31.0.1-jre'
     implementation 'com.opencsv:opencsv:5.6'
```

Comment:
Dumper uses `commons-csv`, using the same might rule-out some issues due to library incompatibility on special/corner cases?

Reply:
Quite the opposite, if lib A has any bug or unexpected behavior, than using in tests lib B allows us to catch that behavior, doesn't it?

Reply:
It will tell where A and B differ in behavior, and both libraries might be just using different - although valid - conventions.

Reply:
Discussed in 1-to-1 chat. No alterations required here. Consider to leave it as is.

Reply:
Can we please share the answer here for other reviewers?

Reply:
Sure,
>Paolo Morandini, Fri 11:22 AM
I'm going to suggest to stick with opencsv and reconsider only if we get some test failures / problems due to incompatibility with commons-csv

Reply:
The assumption is that this would favor coherence with, and re-use of, the already existing test suite for the extraction tool.

Reply:
I think that a test should do exactly what you are saying - enforce the contract. 
When a test is 'flexible' and 'adjusting on the fly' it's very probable to miss a corner case. 

Obviously, there are a bunch of traits of a test, but I think, that a good test should be simple, specific and understandable. So, here we are talking about a test being specific with its data, structure, etc.

I read it as no changes are required here.

Reply:
The contract we aim to enforce is that the loader ends up with a particular set of semantic information, not that a test which selects from the same table obtains the same rows. I think this is the key issue about this entire pull-request chain, and I think this thread is not resolved.